### ISSUE TITLE
Invalid Literal for int() with base 10: 

### ISSUE THREAD

**Issue by Sputn1k7** [hyperspy/hyperspy#2146]
![Metrios3 2_BCF issue](https://user-images.githubusercontent.com/31019888/54550061-8f3bd400-4968-11e9-8485-c7ab64d60090.JPG)

All,
 I am having an issue importing and loading a .bcf file into Hyperspy. The error I'm getting i've attached to this comment.

Has anyone else seen this issue?

**Comment by thomasaarholt** [comment]
A bit more information would be helpful - what type of dataset it is, from which version of which software. Also, it's good practice to paste the actual error message, not just an image of it.

Additionally, try to make your example as small as possible, for instance by loading a single file instead of specifying several with the * symbol. Can you provide the file for debugging?

**Comment by Sputn1k7** [comment]
@thomasaarholt This is a TEM EDS dataset acquired with some commercial automation software (internal version is 3.2)

The folder in question only contains a single .bcf file, the * is just laziness on my part.

**Comment by thomasaarholt** [comment]
@sem-geologist pretty sure this would be the fix. Is there a reason for using `int` on the version number?

```python
>>> int("2.0")
ValueError: invalid literal for int() with base 10: '2.0'
>>> int(float("2.0"))
2
# alternatively
>>> float("2.0")
2.0

**Comment by thomasaarholt** [comment]
@Sputn1k7, if you want a quick-fix, just open `~\AppData\Local\Continuum\Anaconda3\lib\site-packages\hyperspy\io_plugins\bruker.py` in a text editor and replace "int" with "float" on line 583.

**Comment by Sputn1k7** [comment]
I'll give it a shot, thanks @thomasaarholt !

**Comment by Sputn1k7** [comment]
worked great! Does this mean it should be accepted as a fix?

**Comment by sem-geologist** [comment]
that is not correct fix. The problem is that the file version inside xml untill this case was always represented with integer. Also some code in bcf checks for that version (as integer) and outputs additional stuff which is only in version 2 of bcf. I think correct fix would be to do `int(literal_eval(xml_node.text))`.

**Comment by ericpre** [comment]
Fixed in #2146.
